react-native-libraries.json

```diff
@@ -9545,5 +9545,17 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/dozsolti/react-native-use-list",
+    "npmPkg": "react-native-use-list",
+    "nameOverride": "React Native useList",
```

Comment:
Hello @dozsolti, thank you for the contribution to the directory! 👍 

```suggestion
```

`npmPkg` is not needed since the GitHub repository name matches the package name, also when it comes for `nameOverride`, common practice is to stick to the package name, unless there is a good reason not to do it.